Automattic/jetpack
# Test PHPUnit in Travis in parallel

_Testing if it would make sense to run PHPUnit with different WP branches/modes combinations in parallel instead of running them in series._

_Also changed the way we install PHPUnit on older versions of PHP._

### Before
Average time to complete: 10m 16s 
![image](https://user-images.githubusercontent.com/87168/41466782-64ba6312-70ac-11e8-85a9-6363c60e702c.png)

### After
Average time to complete: 5m 50s
![image](https://user-images.githubusercontent.com/7570415/42261131-bdb477c8-7f84-11e8-8a43-b54ee95f8e70.png)


#### Testing instructions:
- TravisCI should spawn 20 parallel jobs
- Unit tests will run on PHP 5.2, 5.6, 7.0 and 7.2
- Lint and misc other tests will run only on PHP 5.6
